### ISSUE TITLE
Can't generate layout files

### ISSUE THREAD

**Issue by y0608** [Eth3rnit3/administrate-tailwind-theme#6]
The command that I run:
```
./bin/rails g administrate_tailwind_theme:views:layout
```

The result:
```
Could not find "../../../app/views/layouts/admin/application.html.erb" in any of your source paths. Your current source paths are: 
/usr/share/rvm/gems/ruby-3.2.2/gems/administrate_tailwind_theme-0.0.5/app/views/administrate/application
```

**Comment by y0608** [comment]
Issue seems to be in layout_generator.rb:
```
def copy_template
  copy_file(
    '../../app/views/layouts/admin/application.html.erb',
    'app/views/layouts/admin/application.html.erb'
  )
end
```

**Comment by Eth3rnit3** [comment]
Indeed, sorry for the inconvenience. I have just corrected, it will be available in the version `0.0.6`

**Comment by y0608** [comment]
No problem! Thank you for maintaining the gem :)
